.circleci/config.yml

```diff
@@ -146,3 +145,50 @@ jobs:
                     --attach build/distributions/git-machete-intellij-plugin-*.zip \
                     --message "$tag"$'\n\n'"$change_notes" \
                     $tag
+  checkForNewEapBuildNumber:
+    docker:
+      - image: 'gitmachete/intellij-plugin-ci:6.1.0'
+    working_directory: ~/git-machete-intellij-plugin
+    resource_class: large
+    steps:
+      - checkout
+      - run:
+          name: Reconfigure origin remote to https
+          command: git remote set-url origin https://${GITHUB_TOKEN}@github.com/VirtusLab/git-machete-intellij-plugin.git
+      - run:
+          name: Configure github user information
+          command: |
+            git config user.email "gitmachete@virtuslab.com"
+            git config user.name "Git Machete Bot"
+      - run:
+          # language=sh
+          command: |
+            ./gradlew updateEapBuildNumber
+            if ! git diff --exit-code --quiet intellijVersions.properties ; then
+              newEapBuildNumber=$(grep -Po '(?<=eapOfLatestSupportedMajor=).*' intellijVersions.properties)
+              branch=update-eap/$newEapBuildNumber
+              git checkout -b "$branch"
+              git add .
+              git commit -m "Add new eap build number $newEapBuildNumber"
+              git push origin "$branch"
+              hub pull-request \
+                --no-edit \
+                --base=develop \
+                --labels=intellij-support
+            fi
+
+workflows:
+  version: 2
+  build:
+    jobs:
+      - build
+  updateEapBuildNumber:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only:
+                - feature/circleCI-job-for-new-eap
+    jobs:
+      - checkForNewEapBuildNumber
```

Comment:
I'd keep the `workflow` and `job` name consistent unless there's a good reason not to